app/models/transformation_mapping/vm_migration_validator.rb

```diff
@@ -128,6 +133,26 @@ def mapped_lans
     @mapped_lans ||= Lan.where(:id => @mapping.transformation_mapping_items.where(:source_type => 'Lan').select(:source_id))
   end
 
+  # rubocop: disable Naming/UncommunicativeMethodParamName
+  def destination_cluster(vm)
+    @mapping.transformation_mapping_items.find_by(:source_type => 'EmsCluster', :source_id => vm.ems_cluster.id).destination
```

Comment:
`:source` is a polymorphic relat so you can just do `.find_by(:source => vm.ems_cluster)`

Reply:
Also probably a good idea to do `&.destination` in case the find_by returns nil

Reply:
Done